CHANGELOG/v1.9.6.md

```diff
@@ -0,0 +1,44 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.28.x -> v1.32.x
+- Workload Cluster: v1.26.x -> v1.32.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.9.5
+## :chart_with_upwards_trend: Overview
+- 13 new commits merged
+- 1 feature addition ✨
+- 3 bugs fixed 🐛
+
+## :sparkles: New Features
+- clusterctl: Add OpenNebula infrastructure provider to clusterctl (#11886)
+
+## :bug: Bug Fixes
+- CABPK: Ensure kubeadm controller always sets all v1beta2 conditions (#11953)
+- ClusterClass: Don't allow concurrent patch upgrades (#11970)
+- Testing: Fix flake TestMachineSetReconciler test (#11917)
+
+## :seedling: Others
+- CAPD: Disable image garbage collection in kubelet to align with kind (#11912)
+- clusterctl: Add support infrastructure provider for Huawei Cloud (#11876)
+- clusterctl: Change k0smotron repo location (#11888)
+- e2e: Log version directly on controller startup (#11964)
```

Comment:
```suggestion
- logging: Log version directly on controller startup (#11964)
```
Sorry my bad, this should have been logging (also just fixed up the label on the PR)